### PR TITLE
Allow Guzzle 7

### PR DESCRIPTION
**Pull request by neoteknic**
Allow Guzzle 7 / Laravel 8

### CODE REVIEW

**Comment by gabrielbull** [comment]
You need to also bump the version in `composer.json`. Looks like no breaking changes so `1.0.1` should do.

**Review by rawbinn — APPROVED** [review]

**Comment by neoteknic** [comment]
Fixed, can you make a new release ?

**Comment by ptondereau** [comment]
@neoteknic Hello, I'm the new maintenair of this package.
One question: Does Laravel 8 require Guzzle 7 a the default main http client?

File: composer.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "php": "^7.1 || ^8.0",
     "ext-dom": "*",
     "ext-simplexml": "*",
-    "guzzlehttp/guzzle": "^6.0",
+    "guzzlehttp/guzzle": "^6.0 || ^7.0",
```
Comment (ptondereau) [inline composer.json:45]: You should add a step in CI for php7.1 with `composer update --prefer-lowest...` to ensure guzzle 6 still work. I can add it if you're ok

**Review by ptondereau — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (neoteknic) [reply composer.json:45]: ok you can add

**Comment by ptondereau** [comment]
@neoteknic thank you1

**Pull request merged**